home-screen-widget/profiles/DEVELOPERS_PORTAL.md

```diff
@@ -0,0 +1,412 @@
+#  TODO
+# Home screen
+The home screen is a feature that surfaces relevant information by displaying a list of widgets in a new main tab on the mobile clients. This gives people effortless access to tools and information that are contextually relevant to their work. 
```

Comment:
Maybe we want to mention that in marketing terms we called the feature "Home", and that that's also what the tab is called on mobile where it shows up.

Reply:
Do you think this should be just the title or should be reflected everywhere? through out the entire document and in the api as well we call it "home screen"

Reply:
This is a technical document, for technical people, so we should call it "home screen" everywhere because that's what we call it in the code. I would just mention here that this is for the feature that is referred to as "Home" or "Home tab" in the marketing material, to avoid confusion.